app/1.0/docs/devguide/data-system.md

```diff
@@ -250,15 +250,15 @@ registered by a wildcard path (for example, you won't see changes to subproperti
 Polymer supports two ways of identifying an array item in a path: by index or by an opaque,
 immutable key.
 
-*   An index, for example, `"myArray.1"` indicates the array item at position 1.
+*   An index, for example, `"myArray.1"`, indicates the array item at position 1.
```

Comment:
You're right that the example here is an appositive, and should be set off as such... But the extra comma makes it a little hard to read. Might be better here to set off the appositives with an em dash, like:

An index—for example, `"myArray.1"`—indicates the... 

That way, the comma after "for example" is unambiguous. 

Also, while you're in there, "indicates" should really be "identifies". I don't know what I was thinking, there.